Lezioni/Argomenti/01_Primi_passi_git/appunti.md

```diff
@@ -0,0 +1,15 @@
+Appunti comandi terminale dal 26/11/2025
+
+- git commit -m= per creare un commit con un messagio.
+- git branch= per vedere la branch.
+- git branch -a= stampa la branch dove mi trovo(verde) e i remotes(rosso).
+- git branch -vv= stampa le informazioni della branch.
+- git fetch --all= va a sincronizzare le nuove informaioni create.
+- git checkout -b= serve per passare da una branch ad un'altra.
+- git push= inserisce la branch nelle etichette.
```

Comment:
git push serve per inviare le modifiche committate in locale sul remote specificato. non so cosa intendi con "nelle etichette"